classes/options/class-discussions.php

```diff
@@ -25,13 +25,20 @@ class DiscussionsPage {
 	 */
 	private $lazy_options;
 
+	/**
+	 * @var string[]
+	 */
```

Comment:
Is this supported by PHP 7.4?

Reply:
I think it is, because my IDE didn't complain (with 7.4 version as target language). But I will remove it for consistency since types are not added in any other attribute.